lib/visualize_packs.rb

```diff
@@ -76,14 +78,14 @@ def self.diagram_title(options, max_violation_count)
     "<<b>#{main_title}</b>#{sub_title}>"
   end
 
-  def self.exclude_packs_info(exclude_packs)
-    case exclude_packs.size
+  def self.to_limited_sentence(list)
+    case list.size
     when 1
-      exclude_packs.first
+      list.first
     when 2
-      exclude_packs.join(" and ")
+      list.join(" and ")
     else
-      "#{exclude_packs[0, 2].join(", ")}, and #{exclude_packs.size - 2} more."
+      "#{list[0, 2].join(", ")}, and #{list.size - 2} more"
```

Comment:
I didn't catch this before, but a unit test for this method would be nice

Reply:
added!